mozregression/download_manager.py

```diff
@@ -234,7 +234,7 @@ def __init__(self, destdir, session=requests, persist_limit=None):
     def get_dest(self, fname):
         return os.path.join(self.destdir, fname)
 
-    def cancel(self, cancel_if=None):
+    def cancel(self, cancel_if=None, wait=False):
```

Comment:
I'm not sure if I understand why this is optional? When would we not want wait to be `True`?